requirements/edx/base.in

```diff
@@ -79,7 +79,7 @@ edx-celeryutils
 edx-completion
 edx-django-release-util             # Release utils for the edx release pipeline
 edx-django-sites-extensions
-edx-django-utils>=3.16.0            # Utilities for cache, monitoring, and plugins; 3.16.0+ for the `pluggable_override`
+edx-django-utils>=4.0.0            # Utilities for cache, monitoring, and plugins
```

Comment:
@timmc-edx: FYI: I followed your README (at first), and this was left as `edx-django-utils>=3.16.0`, which seems odd. Wouldn't I typically want to update this minimum requirement in cases where I am going through the trouble of running `make compile-requirements` to pull in this version in the first place? Maybe some nuance should be added to the README?

Reply:
It did work for me, but if there's already a min constraint in base.in, I'd just update that (and run `make compile-requirements`). Maybe that should actually be the documented process, in fact.

Reply:
@timmc-edx: Agreed. Maybe we could add a note about first checking if there is already a minimum constraint.

Did you want to thumb this, even if conditionally on sandbox testing?